src/bust.py

```diff
@@ -384,19 +384,26 @@ async def send_stats(self, interaction: Interaction) -> None:
                 song_len = 0.0
             submitter_to_len[submit_message.author] += song_len
 
-        # User with longest total submission length
-        longest_submitter = max(submitter_to_len, key=submitter_to_len.get)
-        longest_submitter_time = int(submitter_to_len[longest_submitter])
+        # Format list of users with longest total submission length
+        submitters_sorted_by_len = sorted(
+            [(len, sub) for (sub, len) in submitter_to_len.items()], reverse=True
+        )
+        longest_submitters_formatted = [
+            f"{i+1}. {submitter.mention} - {song_utils.format_time(int(len))}"
+            for i, (len, submitter) in enumerate(
```

Comment:
Note that `len` here is over-shadowing the built-in `len`, which may cause some IDEs to spit out warnings. I'd just name it `length`.